File: backend/BudgetToWealthApi/Models/CashFlow/CashFlowEntryImport.cs
public class CashFlowEntryImport
{
    public required long AmountInCents { get; set; }
    public required DateOnly Date { get; set; }
    public required string CategoryName { get; set; }
    public required CashFlowType CategoryType { get; set; }
    public string? Description { get; set; }
    public RecurrenceFrequency? RecurrenceFrequency { get; set; }
} 

File: frontend/app/import/components/models/CashFlowEntryImport.ts
export type CashFlowEntryImport = {
  amountInCents: number;
  date: string;
  categoryName: string;
  categoryType: "Income" | "Expense";
  description?: string;
  recurrenceFrequency?: "Daily" | "Weekly" | "Monthly" | "Yearly";
} 